tests/Search.test.js

```diff
@@ -123,13 +128,9 @@ describe('Tests for Search', function() {
         });
 
         it('should exist index folder', function() {
-            expect(fs.existsSync(path.join(userConfigDir, 'data', 'search_index_12345678910112'))).toBe(true);
-            expect(fs.existsSync(realTimeIndexPath)).toBe(true);
+            expect(fs.existsSync(path.join(userConfigDir, 'data', 'search_index_12345678910112'))).toBe(false);
```

Comment:
any reason we are hard-coding the number here?

Reply:
This never changes so I have hard-coded it. Also, will be updating the UT in the `R53` release.